src/components/common/icons/index.tsx

```diff
@@ -8,20 +8,18 @@ import Twitter from "./Twitter";
 type IconProps = {
   name: string;
 };
-
 interface iconObject {
   [key: string]: ReactNode;
 }
+const icons: iconObject = {
+  apple: <Apple />,
+  facebook: <Facebook />,
+  logo: <Logo />,
+  menu: <Menu />,
+  twitter: <Twitter />,
+};
 
 const Icon: FC<IconProps> = ({ name }) => {
-  const icons: iconObject = {
-    apple: <Apple />,
-    facebook: <Facebook />,
-    logo: <Logo />,
-    menu: <Menu />,
-    twitter: <Twitter />,
-  };
-
   return <>{icons[name]}</>;
```

Comment:
You can do implicit return here as well :)

Reply:
You're right :see_no_evil:
Updated!